Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -1433,6 +1434,11 @@ namespace AtomSampleViewer
         return true;
     }
 
+    void ScriptManager::Script_SetTestEnvPath(const AZStd::string& envPath)
+    {
+        s_instance->m_envPath = envPath;
```

Comment:
This function really ought to use an operation lambda like the other "Script_" functions do. Otherwise, there can be unexpected behavior. The envPath will be set before all other operations are run. And if a script calls SetTestEnvPath multiple times, only the last call with count, and will apply to any prior uses of CaptureScreenshot.